v5/tutorials/topical/adi.rst

```diff
@@ -139,11 +139,11 @@ Thus an example of use on a lift would look like:
          :caption: autonomous.cpp
          :linenos:
 
-         #define POTENTIOMETER_PORT 1
+         #define POTENTIOMETER_PORT 'A'
          #define MOTOR_PORT 1
 
          void autonomous() {
-           pros::ADIPotentiometer sensor (POTENTIOMETER_PORT);
+           pros::ADIAnalogIn sensor (POTENTIOMETER_PORT);

```

Comment:
this can stay as `pros::ADIPotentiometer`, can't it? it's just a nice alias for `pros::ADIAnalogIn`

Reply:
I see now that @baylessj seemed to imply that it should be changed in #107... can you explain why?

Reply:
This seems to generally be a confusion and consistency issue some people don't understand that there are aliases as mentioned in #106. I thought it would make sense to make these the same everywhere.

Reply:
It was also mentioned in #106 that these were now considered deprecated. Don't know if that is applicable.

Reply:
Ah. In that case, might be good to remove the note about the aliases as well (slow deprecation through invisibility)

Reply:
Are you referring to the issue or is their a note in the docs it self?

Reply:
this 
![image](https://user-images.githubusercontent.com/6646541/73616778-a5d13500-45e5-11ea-9f4d-f70977359659.png)

Reply:
Ok I removed that